jpo-ode-svcs/src/main/resources/application.properties

```diff
@@ -19,18 +19,9 @@ spring.http.multipart.max-request-size=1MB
 #ode.uploadLocationBsm = bsm
 #ode.uploadLocationMessageFrame = messageframe
 
-#USDOT Situation Data Clearinghouse (SDC)/ Situation Data Warehouse (SDW), a.k.a Data Distribution System (DDS) Properties
+#USDOT Situation Data Clearinghouse (SDC)/ Situation Data Warehouse (SDW) Properties

```

Comment:
SDW or SDX?

Reply:
SDX! I have updated non-code references (documentation & comments) from 'SDW' to 'SDX'